addons/website_sale/views/templates.xml

```diff
@@ -195,7 +195,7 @@
                 <div class="container py-2 oe_website_sale">
                     <div class="products_pager form-inline justify-content-center">
                         <t t-call="website_sale.search">
-                            <t t-set="_classes">o_website_sale_search</t>
+                            <t t-set="_classes">o_website_sale_search col-lg-6</t>
```

Comment:
Why ? There is no row here ? In fact do you even need the website_sale.search template ? Should not it be directly a t-call to the search input template ?